exercises/accepting_input/problem.md

```diff
@@ -0,0 +1,48 @@
+# Accepting input
+
+This exercise shows you how to accept input into your service and validate it.
+
+The `context` argument to route handlers holds the information you need about incoming requests to your service, including all data that came along with the request, parsed for use.
+
+- `context.params`: route parameters
+- `context.query`: query string parameters, parsed
+- `context.body`: a promise for a fully-parsed body object
+
+Boltzmann route handlers can have middleware attached to them.
+
+```js
+handlerFunc.route = "GET /"
+hanlderFunc.decorators = [
+    // route-attached middleware goes here
```

Comment:
we may wish to give an example of attaching middleware, or push this section to _after_ the middleware lessons